tests/pdctl/hot/hot_test.go

```diff
@@ -106,15 +104,15 @@ func (s *hotTestSuite) TestHot(c *C) {
 		hotRegion := statistics.StoreHotRegionInfos{}
 		c.Assert(e, IsNil)
 		c.Assert(json.Unmarshal(output, &hotRegion), IsNil)
-		checkInMap.Contains(hotRegion.AsLeader, hotStoreID)
+		c.Assert(hotRegion.AsLeader, HasKey, hotStoreID)
 		c.Assert(hotRegion.AsLeader[hotStoreID].RegionsCount, Equals, 1)
 		c.Assert(hotRegion.AsLeader[hotStoreID].RegionsStat[0].RegionID, Equals, hotRegionID)
 	}
 
 	hotReadRegionID, hotWriteRegionID, hotStoreId := uint64(3), uint64(2), uint64(1)
 	pdctl.MustPutRegion(c, cluster, hotReadRegionID, hotStoreId, []byte("b"), []byte("c"), core.SetReadBytes(1000000000), core.SetReportInterval(reportInterval))
 	pdctl.MustPutRegion(c, cluster, hotWriteRegionID, hotStoreId, []byte("c"), []byte("d"), core.SetWrittenBytes(1000000000), core.SetReportInterval(reportInterval))
-	time.Sleep(3 * time.Second)
+	time.Sleep(3200 * time.Millisecond)
```

Comment:
Why do you change it to `3200 * time.Millisecond`?

Reply:
Because I found that if set to 3s with some local tests, occasionally it may not be able to complete the scheduling and cause the test to fail.